src/Microsoft.OData.Client/BulkUpdateSaveResult.cs

```diff
@@ -139,6 +204,35 @@ internal void BuildDescriptorGraph<T>(IEnumerable<Descriptor> descriptors, bool
             }
         }
 
+        /// <summary>
+        /// Links a descriptor with its link descriptor representation.

```

Comment:
This comment is somewhat confusing. What does "its link descriptor" mean?
I think this was adding a related descriptor, i.e. `linkDescriptor` represents a relationship between the input `descriptor` and the target descriptor of the `linkDescriptor`. Is that the case? If so, maybe the summary and param docs should be adjusted to make that clearer.

Reply:
This was the case for associating a descriptor with its link representation that was added to the entity tracker during a $expand operation.